### PR TITLE
KOGITO-9757 Changed property expansion to use ${property_name}

### PR DESCRIPTION
**Pull request by hbelmiro**
**JIRA:** https://issues.redhat.com/browse/KOGITO-9757

This PR depends on:

- https://github.com/kiegroup/kogito-runtimes/pull/3188

------

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by kaldesai — APPROVED** [review]
@hbelmiro LGTM! Thank you!

**Review by radtriste — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc [hunk 1]
```diff
@@ -212,7 +212,7 @@ For example, consider an application that consumes a resource running on Kuberne
 .Example URI
 [source,shell]
 ----
-org.kie.kogito.sw.knative.service=knative:services.v1/serverless-workflow-greeting-quarkus/greeting-quarkus-cli
+org.kie.kogito.sw.knative.service=${knative:services.v1/serverless-workflow-greeting-quarkus/greeting-quarkus-cli}
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc:215]: Maybe add also some suffix like `/endpoint` to showcase the new feature more?

Comment (radtriste) [reply serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc:215]: @MarianMacik not sure @hbelmiro gonna update that one. Could you propose the change you mean ?

File: serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc [hunk 2]
```diff
@@ -212,7 +212,7 @@ For example, consider an application that consumes a resource running on Kuberne
...
 ----
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc:217]:
```suggestion

Or you can reference a specific endpoint of the Knative service using the following URI:

.Example endpoint URI
[source,shell]
----
org.kie.kogito.sw.knative.service=${knative:services.v1/serverless-workflow-greeting-quarkus/greeting-quarkus-cli}/endpoint
----

```

**Review by MarianMacik — COMMENTED** [review]
Added another example.

File: serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc (hunk 1 above)
Comment (hbelmiro) [reply serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc:215]: Updated. Thank you @MarianMacik.

Comment (radtriste) [reply serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc:215]: thank you @hbelmiro

**Comment by ricardozanini** [comment]
@MarianMacik are we ready?

**Review by MarianMacik — APPROVED** [review]
@ricardozanini Yes, we are.

**Pull request merged**